automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoApiClient.scala

```diff
@@ -359,6 +359,33 @@ object LeonardoApiClient {
         }
     } yield r
 
+  def patchDisk(
+    googleProject: GoogleProject,
+    diskName: DiskName,
+    req: UpdateDiskRequest
+  )(implicit client: Client[IO], authorization: IO[Authorization]): IO[Unit] =
+    for {
+      traceIdHeader <- genTraceIdHeader()
+      authHeader <- authorization
+      r <- client
+        .run(
+          Request[IO](
+            method = Method.PATCH,
+            headers = Headers(authHeader, defaultMediaType, traceIdHeader),
+            uri = rootUri
+              .withPath(Uri.Path.unsafeFromString(s"/api/google/v1/disks/${googleProject.value}/${diskName.value}")),
+            body = req
+          )
+        )
+        .use { resp =>
+          if (!resp.status.isSuccess) {
+            onError(s"Failed to create disk ${googleProject.value}/${diskName.value}")(resp)
```

Comment:
Should this be

```suggestion
            onError(s"Failed to patch disk ${googleProject.value}/${diskName.value}")(resp)
```

?